daemon/src/projection.rs

```diff
@@ -302,10 +314,13 @@ impl Cfd {
                 (CfdState::IncomingSettlementProposal, vec![])
             }
             CollaborativeSettlementCompleted {
-                spend_tx, price, ..
+                spend_tx,
+                price,
+                script,
```

Comment:
I love how this is now also useful here and not just for monitoring. Confirms that packing this information in the event is the right thing to do!

Reply:
Yes, this is nice. The reason I've put a draft originally was myself wondering whether we should pack the script pubkey along the transactions when we can, instead of fishing it out from DLC